src/components/LHNOptionsList/LHNOptionsList.tsx

```diff
@@ -118,11 +118,11 @@ function LHNOptionsList({style, contentContainerStyles, data, onSelectRow, optio
     const renderItem = useCallback(
         ({item: reportID}: RenderItemProps): ReactElement => {
             const itemFullReport = reports?.[`${ONYXKEYS.COLLECTION.REPORT}${reportID}`];
-            const itemParentReport = reports?.[`${ONYXKEYS.COLLECTION.REPORT}${itemFullReport?.parentReportID ?? '-1'}`];
+            const itemParentReport = reports?.[`${ONYXKEYS.COLLECTION.REPORT}${itemFullReport?.parentReportID}`];
             const itemReportNameValuePairs = reportNameValuePairs?.[`${ONYXKEYS.COLLECTION.REPORT_NAME_VALUE_PAIRS}${reportID}`];
             const itemReportActions = reportActions?.[`${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${reportID}`];
             const itemParentReportActions = reportActions?.[`${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${itemFullReport?.parentReportID}`];
-            const itemParentReportAction = itemParentReportActions?.[itemFullReport?.parentReportActionID ?? '-1'];
+            const itemParentReportAction = itemFullReport?.parentReportActionID ? itemParentReportActions?.[itemFullReport?.parentReportActionID] : undefined;
```

Comment:
I used undefined here since that is what is done in this [example](https://github.com/Expensify/App/blob/c8df75fb2cee8a65a505ac458f5c6ac10f981c90/src/components/MoneyRequestHeader.tsx#L66)

Reply:
I can't see the same pattern where you have linked, can you please cross-check and confirm if you have linked correctly?

Reply:
I don't see you doing the same for line 121? Is that because reportaction can be undefined but same is not true for report?

Reply:
Sorry, had the wrong link, it's this one right here: https://github.com/Expensify/App/blob/c8df75fb2cee8a65a505ac458f5c6ac10f981c90/src/components/MoneyRequestHeader.tsx#L66

Reply:
Sorry had the wrong link, it's this one right here:
https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#case-2-type-undefined-cannot-be-used-as-an-index-type

Reply:
<img width="1039" alt="image" src="https://github.com/user-attachments/assets/11e2c229-cb8d-4ec3-a209-c6fea2c43e5f" />

Reply:
Thanks, that looks good to me. 👍